### ISSUE TITLE
Stuck while creating a patch file

### ISSUE THREAD

**Issue by harmankaur23** [insidesherpa/bank-mergeCoTechTask1#162]
"On branch master nothing to commit" 
when i execute 'git commit -m 'Create Patch File'

**Comment by tejas-2232** [comment]
same problem  i encountered

**Comment by Gujar-Shubham** [comment]
same issue
Any solution?

**Comment by tejas-2232** [comment]
actually this problem occurred to me while 2nd attempt. 
In first attempt, the patch was created but not showing in left directory
@Gujar-Shubham what other problems did you encounter

**Comment by Gujar-Shubham** [comment]
yes true i had that first but patch file was not in zip which i downloaded and deleted that file and tried to redo the process then i got this error
actually the patch file is been created and with the usual name and we can also add that with the same usual command process. Thus the issue got cleared

**Comment by tejas-2232** [comment]
But that's seems quite possible...but how you accesed that patch file.
I can't even see it after creating.

**Comment by harmankaur23** [comment]
Yes exactly even I couldn't access the patch file after downloading, I did the whole process again, also try refreshing

**Comment by tejas-2232** [comment]
Ok. I'll try again. Thanks

**Comment by Gujar-Shubham** [comment]
I just assumed that patch file is created and run the next command. And that actually worked for me and i got that idea from a closed issue 55.

**Comment by Cruel-Crab** [comment]
It says nothing to commit because string is not defined by using single quotes in git , 
you need to use double quotes , eg : "Create Patch File"
Hope this works.

**Comment by tejas-2232** [comment]
@harmankaur23 `Refreshing the page worked for me. thanks`
